maintainer/cuda_build.sh

```diff
@@ -0,0 +1,31 @@
+GIT_COMMIT=$(git rev-parse HEAD)
+URL=$(echo "https://gitlab.icp.uni-stuttgart.de/kai/espresso-build/-/jobs/${CI_JOB_ID}")
```

Comment:
nice idea, but thats already done, just forgot to change the script. The script itself is actually just a proof of concept, not a polished version for production testing.